intro.txt

```diff
@@ -506,9 +554,10 @@ steps outline the minimal API calls required to implement an EST proxy:
 
 @section FIPS FIPS 140-2 Compliance
 
-Another consideration is FIPS 140 compliance.  libest was developed
-using OpenSSL 1.0.1.  OpenSSL can be configured and compiled 
-with FIPS 140 support.  This allows libest to claim FIPS compliance when the 
+Another consideration is FIPS 140 compliance.  libEST was developed
+using OpenSSL 1.0.1.  OpenSSL is binary compatible with OpenSSL and 
```

Comment:
The above sentence appears to be incorrect. "OpenSSL is binary compatible with OpenSSL and cane be used with libEST.  This may have been an attempt to clean up references to CiscoSSL.  if it was, this sentence should just go away in the libest version of the code.